packages/paste-core/primitives/text/package.json

```diff
@@ -25,7 +25,7 @@
     "tsc": "tsc"
   },
   "dependencies": {
-    "csstype": "^2.6.10",
+    "csstype": "^3.0.11",
```

Comment:
💡 Same here, we can potentially replace this with `@twilio-paste/style-props` and just use the already declared props from that package.